lib/module.js

```diff
@@ -68,6 +68,10 @@ function httpModule (_moduleOptions) {
     options.retry = 2
   } else if (!options.retry) {
     options.retry = 0
+  } else if (!isNaN(options.retry)) {
+    options.retry = parseInt(options.retry)
+  } else if (typeof options.retry === 'object') {
+    options.retry = JSON.stringify(options.retry)
```

Comment:
Can you please use `null, 2` to format stringified object?

Reply:
Thank you for reviewing.

This code assumes the following:

if options.retry === true
expect `true` tobe `2`

if !options.retry
expect `false` tobe `0`
expect `null` tobe `0`

if !isNaN(options.retry)
expect `2` tobe `2`
expect `"2"` tobe `2`

if typeof options.retry === 'object'
expect `{ limit: 2}` tobe `'{"limit": "2"}'`

else
expect `"null"` tobe  `"null"`